package/kernel/linux/modules/netdevices.mk

```diff
@@ -1223,6 +1223,25 @@ endef
 $(eval $(call KernelPackage,mlx5-core))
 
 
+define KernelPackage/qca8k
+  SUBMENU:=$(NETWORK_DEVICES_MENU)
+  DEPENDS:=@PCI_SUPPORT
```

Comment:
I think this dependency is not necessary.

Reply:
I have removed the dependency.